templates/vue/src/components/tyde/activities/CircleOfSupport/CircleView/index.vue

```diff
@@ -5,15 +5,19 @@
       class="tab"
       :connections="connections"
       :communities="communities"
+      :toolTipPositioned="toolTipPositioned"
       :draggable="!dragDisabled"
       @back="handleBack"
-      @add-connection="$emit('add-connection', $event)"
+      @add-connection="handleConnectionOpen"
+      @connection-opened="handleConnectionOpened"
```

Comment:
"handleConnectionOpen" and "handleConnectionOpened" sound very similar - it's preferable to keep these names more distinct. Couldn't the add connection handler be called "handleAddConnection"? I think that's also more self-explanatory.

Reply:
yes, that one was there before I believe